packages/cli/src/constants/constants.ts

```diff
@@ -1,7 +1,7 @@
 // CLI Server
 export const SERVER_HOST = 'localhost';
 export const SERVER_PORT = 4600;
-export const REDIRECT_ROUTH = '/cli/callback';
+export const REDIRECT_ROUTE = '/cli/callback';
```

Comment:
@djabarovgeorge I think this one caused #271, suggest using the refactor feature on webstorm next time. It changes all the references as well. Right click on the symbol > Refactor > Rename

Reply:
Hey @scopsy yes you are right I wanted to try to make a commit suggestion from Github for the first time, I could not be more wrong lol, I pushed a fix in PR I would love it if you could check it out in your spare time.